.github/workflows/macos.yml

```diff
@@ -1,4 +1,4 @@
-name: MacOS Python build
+name: macos
```

Comment:
Since it's "darwin" in the triple I'm sort of on the fence here. I could go either way.

Reply:
Perhaps the idea with `apple.yml` is that we'd be doing iOS, etc. cross-compiles? We could revisit at that point though.